CWE/events/italy.txt

```diff
@@ -455,6 +455,1450 @@ country_event = {
+				limit = { OR = { culture = italian culture = padanian } }
+				ideology = { factor = 0.03 value = nationalist }
+				militancy = -5
+				consciousness = -2
+			}
+		}
+		
+		734 = { add_province_modifier = { name = terrorist_chaos duration = 183 } } 
+		
+		country_event = 8020327
+		
+		ai_chance = { factor = 0.2 }
+	}
+
+}
+
+#Alto-Adige's Long Summer (AUS Answer)
+country_event = {
+  	id = 8020325
+  	title = "The Status of Germans in South Tyrol/Alto-Adige"
+  	desc = EVT_8020325_DESC
+  	picture = "nwo2_austrian_tyrol_1972"
+	
+  	is_triggered_only = yes
+
+	option = {
+		name = "Guarantee for the Germans in Italy (War Risk)"
+		badboy = 1
+		
+		prestige = 10
+		
+		AUS_734 = {
+			any_pop = { 
+				limit = { OR = { culture = italian culture = padanian } }
+				scaled_militancy = { factor = 4 ideology = nationalist }
+                scaled_consciousness = { factor = 2 ideology = nationalist }
+				militancy = 2
+				consciousness = 1
+			}
+		}
+		
+		random_country = {
+                       limit = { tag = ITA is_vassal = no }
+                       relation = { who = THIS value = -25 }
+        }
+		
+		ITA = { country_event = 8020326 } 
+		
+		ai_chance = { factor = 0.5 }
+	}
+	
+	option = {
+		name = "Do not interfere in Italian business"
+		prestige = -10
+		
+		AUS_734 = {
+			any_pop = { 
+				limit = { OR = { culture = german culture = austrian_german } }
+				ideology = { factor = 0.02 value = nationalist } 
+				militancy = 4
+				consciousness = 2
+			}
+			any_pop = { 
+				limit = { OR = { culture = italian culture = padanian } }
+				militancy = -5
+				consciousness = -3
+			}
+		}
+		
+		random_country = {
+                       limit = { tag = ITA is_vassal = no }
+                       relation = { who = THIS value = 25 }
+        }
+		
+		ai_chance = { factor = 0.5 }
+	}
+
+}
+
+#Alto-Adige's Long Summer (ITA Answer)
+country_event = {
+  	id = 8020326
+  	title = "Austria's Ultimatum"
+  	desc = EVT_8020326_DESC
+  	picture = "1961_feuernacht"
+	
+  	is_triggered_only = yes
+
+	option = {
+		name = "Form the Autonomous Province"
+		badboy = 1
+		
+		prestige = 10
+		
+		AUS_734 = {
+			any_pop = { 
+				limit = { OR = { culture = italian culture = padanian } }
+				scaled_militancy = { factor = 4 ideology = nationalist }
+                scaled_consciousness = { factor = 2 ideology = nationalist }
+				militancy = 2
+				consciousness = 1
+			}
+		}
+		
+		random_country = {
+                       limit = { tag = ITA is_vassal = no }
+                       relation = { who = THIS value = -25 }
+        }
+		
+		ai_chance = { factor = 0.9 }
+	}
+	
+	option = {
+		name = "Make no concession to the Austrians"
+		badboy = 5
+		
+		FROM = { relation = { who = THIS value = -100 } }
+		
+		FROM = { 
+		    war = {							
+			    target = ITA
+			    attacker_goal = { casus_belli = acquire_any_state state_province_id = 735 }
+			    defender_goal = { casus_belli = status_quo  } 
+			    call_ally = no
+		    }
```

Comment:
The event that triggers this event chain must check that Austria is independent as this event triggers a war with Italy.